site/content/docs/5.1/components/spinners.md

```diff
@@ -38,14 +44,39 @@ The border spinner uses `currentColor` for its `border-color`, meaning you can c
 <div class="spinner-border text-secondary" role="status">
   <span class="visually-hidden">Loading...</span>
 </div>
+<div class="spinner-border text-white" role="status">
+  <span class="visually-hidden">Loading...</span>
+</div>
 {{< /example >}}
 <!-- End mod -->
 
 {{< callout info >}}
 **Why not use `border-color` utilities?** Each border spinner specifies a `transparent` border for at least one side, so `.border-{color}` utilities would override that.
 {{< /callout >}}
 
-<!-- Boosted: no Growing spinner -->
+## Growing spinner
+
+If you don't fancy a border spinner, switch to the grow spinner. While it doesn't technically spin, it does repeatedly grow!
+
+{{< example >}}
+<div class="spinner-grow" role="status">
+  <span class="visually-hidden">Loading...</span>
+</div>
+{{< /example >}}
+
+Once again, this spinner is built with `currentColor`, so you can easily change its appearance with [text color utilities][color]. Here it is in blue, along with the supported variants.
+
+{{< example >}}
+<div class="spinner-grow text-primary" role="status">
+  <span class="visually-hidden">Loading...</span>
+</div>
+<div class="spinner-grow text-secondary" role="status">
+  <span class="visually-hidden">Loading...</span>
+</div>
+<div class="spinner-grow text-light" role="status">
```

Comment:
For `.text-light`, we override the background-color and it doesn't appear as white. Should we use bg-color utilities instead ?

Reply:
IMO we can drop this example since it is not useful with a dark background